docs/design/coding-conventions.md

```diff
@@ -314,13 +314,58 @@ Note that this also applies to uses of "ID" as an abbreviation for "identifier"
 
 Prefixes based on types (e.g., `p` for pointers) should never be used.
 
-Global variables should have a `g` prefix, e.g. `gCounter`.
-Non-`const` `static` class members can have an `s` prefix if that suits your fancy.
+Global variables should have a `g_` prefix, e.g. `g_counter`.
+Non-`const` `static` class members can have an `s_` prefix if that suits your fancy.
 Of course, both of these should be avoided, so this shouldn't come up often.
 
 Constant data (in the sense of `static const`) should have a `k` prefix.
```

Comment:
Is it an oversight that you don't propose moving `k` to `k_`?

Reply:
No. I only suggest g_ and s_ as it is consistent with general practice around m_. As pointed out in the text the s g cases are somewhat rare - so is arguably not that important, certainly not a show stopper *against* m_. If you wanted gBlah and m_. I mean that's odd, but is not a show stopper.